integration-tests/data/instructions-with-bytes-contracts.ts

```diff
@@ -0,0 +1,9 @@
+import fs from 'fs';
+import path from 'path';
+
+export const addContract =  fs.readFileSync(path.resolve(`${__dirname}/../../packages/taquito-michel-codec/test/contracts_016/opcodes/and_bytes.tz`)).toString();
```

Comment:
I'm not sure why we would need fs for this. You should be able to do something like this https://github.com/ecadlabs/taquito/blob/master/integration-tests/data/allTestsCases.ts#L35

Reply:
I tried that but with the `.tz` file it doesnt seem like it can be imported like that